src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/create_index_pattern_wizard.tsx

```diff
@@ -222,9 +254,9 @@ export class CreateIndexPatternWizard extends Component {
           isIncludingSystemIndices={isIncludingSystemIndices}
           esService={services.es}
           savedObjectsClient={services.savedObjectsClient}
-          indexPatternCreationType={this.indexPatternCreationType}
+          indexPatternCreationType={indexPatternCreationType}
           goToNextStep={this.goToTimeFieldStep}
-          uiSettings={services.uiSettings}
+          uiSettings={services.config}
```

Comment:
Could you please rename config -> uiSettings

Reply:
done